index.js

```diff
@@ -20,37 +23,35 @@ module.exports = {
         'No Applitools API key found! Set APPLITOOLS_API_KEY with your API key from https://eyes.applitools.com',
       );
     }
-    
-    const applitoolsConfig = {
-      showLogs: true
-    }
-    
-    fs.writeFile(path.resolve(PUBLISH_DIR, '..', 'applitools.config.js'), `module.exports = ${JSON.stringify(applitoolsConfig)}`, 'utf8');
```

Comment:
`fs.writeFile` is an `async` method so there's an `await` missing here.

I moved that line a bit further down the execution path so it can be executed in parallel